### PR TITLE
add DQM Validation plots for MSSM Hbb Trigger Paths.

### PR DESCRIPTION
**Pull request by heppye**
add DQM Validation plots for MSSM Hbb Trigger Paths.

### CODE REVIEW

**Comment by heppye** [comment]
this PR have been checked by STEAM experts already.  Information could be found at https://its.cern.ch/jira/browse/CMSHLT-470. Please process if possible.

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
+1

File: HLTriggerOffline/Higgs/python/hltHiggsPostProcessors_cff.py [hunk 1]
```diff
@@ -320,6 +320,29 @@ def get_reco_strings(strings):
 hltHiggsPostWHToENuBB.subDirs = ['HLT/Higgs/WHToENuBB']
 hltHiggsPostWHToENuBB.efficiencyProfile = efficiency_strings_WHToENuBB
 
+#Specific plots for MSSMHbb
+#Jet plots
+NminOneCuts = (_config.__getattribute__("MSSMHbb")).__getattribute__("NminOneCuts")
```
Comment (davidlange6) [inline HLTriggerOffline/Higgs/python/hltHiggsPostProcessors_cff.py]: would not like to use this backdoor for the config - why can the the pset not just be imported?

**Comment by heppye** [comment]
@davidlange6 , thanks for taking time to do the careful check in such a short notice. I actually following the coding convention of previous Hbb trigger paths. As I am kind of new to DQM and the time line is really tight, I just try to do  as minimium modification to the existing code as possible. Please confirm whether current modification is suffient for monitoring.

**Comment by davidlange6** [comment]
@heppye  - you can better do something like this:

from HLTriggerOffline.Higgs.hltHiggsValidator_cfi import hltHiggsValidator
NMinOneCuts = hltHiggsValidator.MSSMHbb.NminOneCuts

**Comment by heppye** [comment]
@davidlange6 , I was actually just testing the line locally. However, does it mean redo the dqm and build test again?

**Comment by heppye** [comment]
@davidlange6  ok,  I will commit the change soon ...

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
ciao @heppye 
at this point could you change the rest of the code for consistency?
moreover please avoid leaving commented lines in prod code. referring to:
https://github.com/heppye/cmssw/blob/CMSSW_7_6_X/HLTriggerOffline/Higgs/python/hltHiggsPostProcessors_cff.py#L327

thanks,
F.

**Comment by davidlange6** [comment]
cmsbuild, please test

**Comment by heppye** [comment]
@deguio  thanks for the comments. The cmsbuild test was already start when I saw your massage...

**Comment by heppye** [comment]
@deguio, the updates were done to the rest code into the style suggested by @davidlange6

**Comment by heppye** [comment]
@jasperlauwers @silviodonato  add people whom have been worked on those files for steering...

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
thanks @heppye

**Comment by heppye** [comment]
@deguio @davidlange6  test are fine, please process if possible.

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**